test/commands/convert/dbprotect2hdf.spec.ts

```diff
@@ -1,5 +1,5 @@
-/* eslint-disable array-bracket-newline */
-/* eslint-disable array-element-newline */
+
```

Comment:
I will hold from commenting in each situation. Whenever we update to remove the extra newlines, we can do a sweep to check for all of them.